SparkPost/2web2ui
# SA-856 - Allow users to open the links in the various reports pages' table in new tabs

### What Changed
 - Added a href to the summary table. 
 - Stop the page from redirecting if just adding a new filter on the existing page.

### How To Test
 -Go to summary reports under reports/summary. 
 - Change the group by in the table to another filter type.
 - Click on the filter link. It should add another filter without refreshing the page.
 - Verify that you can right click on the filter and open a new tab as well.
